src/constants/style.scss

```diff
@@ -24,12 +23,34 @@ $color-pink: #F20574;
 
 // board & column constants
 $board__side-panel-width: 64px;
-
 $column__min-width: 320px;
 $column__max-width: 960px;
 $column__border-width: 12px;
 
 
+// margin and padding
+$small-padding: 16px;
+$medium-padding: 20px; 
+
+
+// font
+$small-letter-spacing: 0.25px;
```

Comment:
I think inverting the names of the variables will be handy here. So writing `letter-spacing-...` instead of `...-letter-spacing` will make it easier to use the auto complete features of the IDE. Also I would suggest to use `--` to separate the modifier and write `letter-spacing--small` etc.

The same naming scheme should apply on the other variables.

Reply:
Done